src/test/java/uk/gov/justice/digital/zone/Fixtures.java

```diff
@@ -16,15 +15,15 @@ public class Fixtures {
 
     public static final String TABLE_SOURCE = "oms_owner";
     public static final String TABLE_NAME = "agency_internal_locations";
-    public static final String OPERATION = "load";
+    public static final String TABLE_OPERATION = "load";
```

Comment:
one general comment. Can we move this Fixtures.java to test.helpers folder?

Reply:
The idea is that they're specific to the zone tests which is why they are in this location. Also they're fixtures rather than helpers in this instance.